bucket/bugn.json

```
@@ -0,0 +1,31 @@
+{
+    "homepage": "https://github.com/fuhsjr00/bug.n",
+    "description": "bug.n is a tiling window manager add-on for the Explorer shell of Microsoft Windows. It is written in the scripting language AutoHotkey",
+    "license": {
```

- Wrong SPDX identifier.
- Do not use URL when it is SPDX license

```suggestion
    "license": "GPL-3.0-or-later",
```

---

Thanks. Done.